lib/src/sync/common.dart

```diff
@@ -16,6 +16,7 @@ import 'web_driver.dart';
 import 'web_element.dart';
 
 const String elementStr = 'ELEMENT';
+const String w3cElementStr = 'element-6066-11e4-a52e-4f735466cecf';
```

Comment:
Can you leave a comment in the code right there with this URL? It's way too magic.